addons/mail/static/src/core/common/composer.js

```diff
@@ -808,13 +816,34 @@ export class Composer extends Component {
         }
     }
 
-    restoreContent() {
+    async restoreContent() {
         const composer = toRaw(this.props.composer);
         try {
             const config = JSON.parse(browser.localStorage.getItem(composer.localId));
             if (config.text) {
                 composer.emailAddSignature = config.emailAddSignature;
                 composer.text = config.text;
+                await this.suggestion.suggestionService.fetchPartners(
+                    config.mentionedPartners
+                        .filter((m) => !this.store.Persona.get(m))
+                        .map((m) => m.id),
+                    this.thread
+                );
+                this.props.composer.mentionedPartners =
+                    this.suggestion.suggestionService.filterPartnerSugestions(
+                        config.mentionedPartners.map((mentionedPartner) =>
+                            this.store.Persona.get(mentionedPartner)
+                        )
+                    );
```

Comment:
I don't see a point to filter partner suggestions: this is too verbose compared to filtering there immediately with explicit code (same as other comments), but also the `fetchPartners` necessarily fetch the names...

Reply:
@seb-odoo asked that the restored mention be filtered the same way mentions are filtered. 
I do not think multiple filters are a good solution.

Reply:
Maybe a good compromise is to turn `filterPartnerSugestions` as a stand-alone function, these LOCs become easier to read as `filterPartnerSugestions(...)` than `this.suggestion.suggestionService.filterPartnerSugestions(...)`.

Reply:
But this filter doesn't have any usefulness outside of mentions.

> These LOCs become easier to read as filterPartnerSugestions(...) than this.suggestion.suggestionService.filterPartnerSugestions(...).

No, it's just shorter, not easier. In the future, you are going to look at this code and ask "Who's the idiot who created a stupid filter function out of nowhere". Unless you know it's part of the suggestion service, it makes no sense.

Reply:
The function doesn't use `this`, and the function is named in a way that is already self-sustained, therefore there's not much reason to store in the service. Shorter code leads quite often to simpler code, here the code is complex because of long accessor chains.